src/server/trpc.ts

```diff
@@ -0,0 +1,10 @@
+// this file is used to help bring in some of the important tools from @trpc, used to create routers,
+// whereas index.ts is where we actually use these things to assemble the main router itself
+// (assemble routers representing different namespaces of concepts within the app, for example)
```

Comment:
address comments, make changes where necessary. If the docs are a more appropriate place to put this (it probably is), put it there.